### ISSUE TITLE
Performance report and Simple model comparison don't show graph for binary str label

### ISSUE THREAD

**Issue by nirhutnik** [deepchecks/deepchecks#491]
![image](https://user-images.githubusercontent.com/92314933/148099793-01ee0a05-b5ab-44d6-ac42-26b12ebcc2bf.png)

**Comment by nirhutnik** [comment]
Code to recreate:

```
import traceback

import numpy as np
import pandas as pd
from catboost import CatBoostClassifier

pd.set_option('display.max_columns', None)

cols = ['age',
'class of worker',
'detailed industry recode',
'detailed occupation recode',
'education',
'wage per hour',
'enroll in edu inst last wk',
'marital stat',
'major industry code',
'major occupation code',
'race',
'hispanic origin',
'sex',
'member of a labor union',
'reason for unemployment',
'full or part time employment stat',
'capital gains',
'capital losses',
'dividends from stocks',
'tax filer stat',
'region of previous residence',
'state of previous residence',
'detailed household and family stat',
'detailed household summary in household',
'migration code-change in msa',
'migration code-change in reg',
'migration code-move within reg',
'live in this house 1 year ago',
'migration prev res in sunbelt',
'num persons worked for employer',
'family members under 18',
'country of birth father',
'country of birth mother',
'country of birth self',
'citizenship',
'own business or self employed',
'fill inc questionnaire for veteran\'s admin',
'veterans benefits',
'weeks worked in year',
'year','target']

data =  pd.read_csv('c://users/nirhu/Projects/Bug bash/census-income.data', header=None)
test =  pd.read_csv('c://users/nirhu/Projects/Bug bash/census-income.test', header=None)
data.drop(columns=31, inplace=True)
test.drop(columns=31, inplace=True)

data.columns = cols
test.columns = cols

model = CatBoostClassifier()

sample_train = data.sample(1000)
sample_test = test.sample(1000)
x_train, y_train = sample_train.drop(columns='target'), sample_train['target']
x_test, y_test = sample_test.drop(columns='target'), sample_train['target']

cat_cols = ['class of worker',
'detailed industry recode',
'detailed occupation recode',
'education',
'enroll in edu inst last wk',
'marital stat',
'major industry code',
'major occupation code',
'race',
'hispanic origin',
'sex',
'member of a labor union',
'reason for unemployment',
'full or part time employment stat',
'tax filer stat',
'region of previous residence',
'state of previous residence',
'detailed household and family stat',
'detailed household summary in household',
'migration code-change in reg',
'migration code-move within reg',
'live in this house 1 year ago',
'migration prev res in sunbelt',
'num persons worked for employer',
'country of birth father',
'country of birth mother',
'country of birth self',
'citizenship',
'own business or self employed',
'fill inc questionnaire for veteran\'s admin',
'veterans benefits',
'year']

model.fit(x_train, y_train, cat_features=cat_cols, verbose=0)
from deepchecks import Dataset
from deepchecks.suites import full_suite

fs = full_suite()

ds_train = Dataset(sample_train, label='target', cat_features=cat_cols)
ds_test = Dataset(sample_test, label='target', cat_features=cat_cols)

fs.run(ds_train, ds_test, model)
```
